docs/UserGuide.md

```diff
@@ -211,6 +211,14 @@ HeliBook data are saved as a JSON file `[JAR file location]/data/addressbook.jso
 If your changes to the data file makes its format invalid, HeliBook will discard all data and start with an empty data file at the next run.
 </div>
 
+### Exporting and importing your data
```

Comment:
Would be good if you could separate the export and import into 2 different parts.